### ISSUE TITLE
Support Altair development environment

### ISSUE THREAD

**Issue by prudhomm** [feelpp/feelpp#776]
/cc @feelpp/po

**Comment by prudhomm** [comment]
@vincentchabannes the `feelpp_database_*` have been installed in `feelpp-libs`. Could you test if it works properly ? could you add to feel++ the acusim data @StephieEDWIGE produced to test the tools ?

**Comment by prudhomm** [comment]
@feelpp/po the conversion, pod and export tools are now distributed via docker

**Comment by vincentchabannes** [comment]
@prudhomm I have this error in the docker container for all applications : 
```
feelpp_qs_laplacian_2d: error while loading shared libraries: libh3dreader.so: cannot open shared object file: No such file or directory
```

**Comment by prudhomm** [comment]
I fixed that in #895, I will rebuild as soon as I get the automatic dockerization ready (I am close now I think)
